server/routes/utils.js

```diff
@@ -52,18 +52,18 @@ exports.errorHandler = (err, req, res, next) => {
   }
 };
 
-exports.apiKeyHandler = exports.handlerWrapper(async (req, res, next) => {
-  const session = new Session();
-  const apiKey = new ApiKeyService(session);
-  await apiKey.check(req.headers['treetracker-api-key']);
-  log.debug('Valid Access');
-  next();
-});
+// exports.apiKeyHandler = exports.handlerWrapper(async (req, res, next) => {
```

Comment:
Good.  Don't worry about this or the organization ID for now.  We will only expose this tool to one organization using the permissions system at first, as we await the deployment of keycloak.